backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/storage/generated/tables/AiCodeFixSettingsTable.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SonarLint Core - Commons
+ * Copyright (C) 2016-2025 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+/*
+ * Minimal stub of a jOOQ generated Table for AI_CODEFIX_SETTINGS.
+ * This is only to make the project compile in environments where the jOOQ
+ * code generation step is not executed. A normal Maven build should generate
+ * a richer version of this class.
+ */
+package org.sonarsource.sonarlint.core.commons.storage.generated.tables;
+
+import java.sql.Timestamp;
+import org.jooq.Record;
+import org.jooq.TableField;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.jooq.impl.TableImpl;
+
+public final class AiCodeFixSettingsTable extends TableImpl<Record> {
```

Comment:
I wonder if we need to put these under source control. We could generate them at build time, e.g. using a container to run the DB